lib/autoyast.pm

```diff
@@ -440,7 +440,9 @@ sub adjust_network_conf {
 sub expand_variables {
     my ($profile) = @_;
     # Expand other variables
-    my @vars = qw(SCC_REGCODE SCC_REGCODE_HA SCC_REGCODE_GEO SCC_REGCODE_HPC SCC_REGCODE_WE SCC_URL ARCH LOADER_TYPE NTP_SERVER_ADDRESS SCC_REGCODE_LTSS);
+    my @vars = qw(SCC_REGCODE SCC_REGCODE_HA SCC_REGCODE_GEO SCC_REGCODE_HPC
+      SCC_REGCODE_WE SCC_URL ARCH LOADER_TYPE NTP_SERVER_ADDRESS
+      SCC_REGCODE_LTSS REPO_SLE_MODULE_DEVELOPMENT_TOOLS);
```

Comment:
Can i propose to replace this with ADDONS or ADDONURL? the ADDONS should be equal to sdk, i think. for instance (ADDONS=sdk). I know that i dont make it easy for you but the reason is that it will make the expanding extensible. Apparently it will require some changes all around. I totally get it if you dont want to do that in this PR so i recommend to open an issue for discussion and file a progress ticket. sounds good? or do you think that it does not make sense?

Reply:
I don't get your suggestion here.

Reply:
I'm only reusing existing autoyast method to replace variables that we need here, I need this exact one to not try to construct its value and just getting from the setting of the current build. There is `console/validate_addon_repos.pm` which use those variables and probably this job should look more like that one, but we might want to do some other things with this job int the future apart from installing addon so for my case works fine like that. If you have further proposal don't hesitate on creating some ticket to unify.